### PR TITLE
chore (samples) Refactoring: added invalid catalog name in import product from gcs.

### PR DESCRIPTION
**Pull request by dfirova**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-retail/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

If you write sample code, please follow the [samples format](
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/SAMPLE_FORMAT.md).

### CODE REVIEW

File: samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java [hunk 1]
```diff
@@ -44,6 +44,10 @@ public static void main(String[] args) throws IOException, InterruptedException
     String branchName =
         String.format(
             "projects/%s/locations/global/catalogs/default_catalog/branches/0", projectId);
+    // TO CHECK ERROR HANDLING USE INVALID CATALOG IN REQUEST
+    // branchName =
+    // String.format("projects/%s/locations/global/catalogs/invalid_catalog/branches/default_branch", PROJECT_NUMBER);
```
Comment (chenlei1216) [inline samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java:49]: projectId?

**Review by chenlei1216 — APPROVED** [review]

File: samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java [hunk 2]
```diff
@@ -44,6 +44,10 @@ public static void main(String[] args) throws IOException, InterruptedException
     String branchName =
         String.format(
             "projects/%s/locations/global/catalogs/default_catalog/branches/0", projectId);
+    // TO CHECK ERROR HANDLING USE INVALID CATALOG IN REQUEST
```
Comment (alicejli) [inline samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java:47]: nit: can this be made into lower case? e.g. `To check error handling, use an invalid catalog in request` or something similiar

**Review by averikitsch — APPROVED** [review]

File: samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java (hunk 1 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java]: Refactored.

File: samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java (hunk 2 above)
Comment (dfirova) [reply samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java]: Refactored.

**Review by alicejli — APPROVED** [review]

**Pull request merged**